src/Symfony/Component/JsonEncoder/Decode/Instantiator.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonEncoder\Decode;
+
+use Symfony\Component\JsonEncoder\Exception\UnexpectedValueException;
+
+/**
+ * Instantiates a new $className eagerly, then set the given properties.
```

Comment:
```suggestion
 * Instantiates a new $className eagerly, then sets the given properties.
```

![image](https://github.com/user-attachments/assets/37cbc88d-1996-4f0e-ae89-8012bdcb9de1)